convert2rhel/actions/pre_ponr_changes/subscription.py

```diff
@@ -214,27 +216,28 @@ def run(self):
             logger.task("Convert: Subscription Manager - Enable RHEL repositories")
             subscription.enable_repos(rhel_repoids)
         except OSError as e:
-            # TODO(r0x0d): Places where we raise OSError and need to be
-            # changed to something more specific.
-            #  - Could fail in invoking subscription-manager to get repos     (get_avail_repos)
-            #  - Could fail in invoking subscirption-manager to disable repos (disable_repos)
-            #  - ""                                          to enable repos  (enable_repos)
+            # This should not occur anymore as all the relevant OSError has been changed to a CriticalError
+            # exception. This exception handler is just a precaution
```

Comment:
Just to point out, when I did this exception block back then my rationale was something like: "I don't know if that will even happen, but in case it does, we have a way to catch it... Being extra paranoid, I know..." 

So I can't really confirm if that really happens.